docs/design/automatic-updates.md

```diff
@@ -0,0 +1,38 @@
+
+# Storage Node automatic updates DesignDocument
+
+## Overview
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date. If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.
+
+## Goals
+
+Create an automatic update system into the Storage Node software so that node operators do not have to worry about updating their nodes.
+Keep the Storage Nodes on the network up to date with the latest software to improve stability and performance. 
+Concerns
+
+A majority or substantial amount of nodes update their software at the same time causing instability in the network or temporary file lose.
+We must ensure the automatic update system we build into the Storage nodes works on ALL OS we support.
+
+## Scenarios
+
+
+## Design Overview
+
+When starting a storj node:
+* Main process starts an updater service which starts a (12 hour?) interval loop
```

Comment:
Should this design doc cover OS specific auto-startup of the updater service?  We'd be losing that compared to Docker.

Reply:
It's a binary that you run... not sure how it would differ by OS at this level

Reply:
Download folder, services, anti-virus, firewalls.

Reply:
This is much more than a 3 point story then.